package.json

```diff
@@ -4650,6 +4663,16 @@
                     "when": "!gitlens:readonly && viewItem =~ /gitlens:remotes\\b/",
                     "group": "1_gitlens@1"
                 },
+                {
+                    "command": "gitlens.views.pruneRemote",
+                    "when": "!gitlens:readonly && viewItem =~ /gitlens:remote\\b/",
+                    "group": "inline@97"
+                },
+                {
+                    "command": "gitlens.views.pruneRemote",
```

Comment:
Can you move this down to be right after `gitlens.views.fetch`, and then change the group to be `1_gitlens@2` so that way `Fetch` will still be the first command on the context-menu